portfolio/src/main/webapp/style.css

```diff
@@ -11,31 +11,33 @@ body {
   padding: 1%;
 }
 
-#intro-content {
-  margin-left: auto;
-  margin-right: auto;
-  text-align:center;
-  width: 650px;
-}
-
 #basic-profile {
    border: 1px solid black;
+   display:inline-block;
    padding: 10px;
 }
 
-#inner-content {
-   border: 1px solid black;
-   text-align: center;
+form {
+  display:inline-block;
 }
 
-#intro-buttons {
-  display: inline-block;
-  padding: 1%;
+#icon {
+  display:inline-block;
```

Comment:
[style nit](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops) - and with all of the property names in this file:
> Always use a single space between property and value

Reply:
thanks! updated!